tests/integration/test_fim/test_files/test_basic_usage/test_basic_usage_realtime_unsupported.py

```diff
@@ -0,0 +1,73 @@
+# Copyright (C) 2015-2021, Wazuh Inc.
+# Created by Wazuh, Inc. <info@wazuh.com>.
+# This program is free software; you can redistribute it and/or modify it under the terms of GPLv2
+
+import os
+
+import pytest
+import re
+from wazuh_testing import global_parameters
+from wazuh_testing.fim import generate_params, regular_file_cud, LOG_FILE_PATH, callback_num_inotify_watches, \
+                              detect_initial_scan, callback_ignore_realtime_flag, CHECK_ALL, REQUIRED_ATTRIBUTES
+from wazuh_testing.tools import PREFIX
+from wazuh_testing.tools.configuration import load_wazuh_configurations, check_apply_test
+from wazuh_testing.tools.monitoring import FileMonitor
+
+# Marks
+
+
+pytestmark = [pytest.mark.darwin, pytest.mark.sunos5, pytest.mark.tier(level=0)]
+
+# variables
+
+
+directory_str = os.path.join(PREFIX, 'dir')
+test_data_path = os.path.join(os.path.dirname(os.path.realpath(__file__)), 'data')
+configurations_path = os.path.join(test_data_path, 'wazuh_conf_check_realtime.yaml')
+test_file = "testfile.txt"
```

Comment:
```suggestion
test_file = 'testfile.txt'
```

Reply:
Done in [commit](https://github.com/wazuh/wazuh-qa/pull/1474/commits/1b7ea0a207fdaf1ba67ee56d13785ac575b1ae0a)